operator/cmd/mesh/root.go

```diff
@@ -24,9 +24,10 @@ import (
 )
 
 const (
-	SetFlagHelpStr = `Set a value in IstioOperator CustomResource. e.g. --set policy.enabled=true.
-Overrides the corresponding path value in the selected profile or passed through IstioOperator CR
-customization file`
+	SetFlagHelpStr = `Can be used to override a path in the IstioOperator API spec, e.g. to choose a profile
```

Comment:
Simplify grammar: `Override an IstioOperator value, e.g. ...`

Reply:
That rolls of the tongue much better! Done.

Reply:
I added the `Example:` to the cobraCommands for _manifest-apply.go_, however was unable to find any related commands that make use of `--set`.  Let me know if there is anything else I'm missing.